### PR TITLE
python3Packages.tubeup: unbreak 

### PR DESCRIPTION
**Pull request by freezeboy**
###### Motivation for this change

Unbreak tubeup python module. It seems there are two different youtube-dl(c) packages.
The selected one was not the correct one

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/youtube-dlc/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+buildPythonPackage rec {
+  pname = "youtube_dlc";
+  version = "2020.11.11.post3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "WqoKpfvVPZrN+pW6s8JoApJusn5CXyPcg9VcsY8R0FM=";
+  };
+
+  outputs = [ "out" "dev" ];
```
Comment (jonringer) [inline pkgs/development/python-modules/youtube-dlc/default.nix:12]:
```suggestion
```

common for python packages to use this to pass runtime packages
```
cat ./result-dev/nix-support/propagated-build-inputs
/nix/store/18656kvqazm74bj7k3mdkwmdlqfyf581-python3-3.8.6  /nix/store/wwsf5czpcmxbqzqv5p623cc40jdn6cg0-python3.8-youtube_dlc-2020.11.11.post3
```

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 104222` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>5 packages built:</summary>
  <ul>
    <li>python27Packages.youtube-dlc</li>
    <li>python37Packages.tubeup</li>
    <li>python37Packages.youtube-dlc</li>
    <li>python38Packages.tubeup</li>
    <li>python38Packages.youtube-dlc</li>
  </ul>
</details>

File: pkgs/development/python-modules/youtube-dlc/default.nix (hunk 1 above)
Comment (doronbehar) [reply pkgs/development/python-modules/youtube-dlc/default.nix]: That's solved.

**Review by doronbehar — APPROVED** [review]

**Comment by doronbehar** [comment]
> Result of `nixpkgs-review pr 104222` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)

Thanks @SuperSandro2000 .

**Pull request merged**